packages/components/src/utils/AppearanceAnim.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from "react"
+import cx from "classnames"
```

Comment:
Nit: Easier reference when reading
```suggestion
import classnames from "classnames"
```